docusaurus.config.ts

```diff
@@ -15,6 +15,7 @@ const config: Config = {
   tagline:
     "This toolkit will guide you on implementing secure and user-friendly digital and in-person interactions. Whether you are a service owner, developer or somewhere in between; use it to navigate your adoption journey with practical workshops, templates and guides.",
   favicon: "img/favicon.ico",
+  staticDirectories: ['contexts', 'static'],
```

Comment:
Might want to consider a global directory called `machine_readable_governance` that has these as static subfolders

Reply:
agreed